test/integration/common

```diff
@@ -83,6 +83,25 @@ check-synced-entry() {
   fail-now "timed out waiting for agent to sync down entry"
 }
 
+check-svid-count() {
+  MAXCHECKS=50
+  CHECKINTERVAL=1
+
+  for ((i=0;i<=MAXCHECKS;i++)); do
+    if (( $i>=$MAXCHECKS )); then
+      fail-now "svid count validation failed"
```

Comment:
```suggestion
      fail-now "X.509-SVID count validation failed"
```
1. This `if` condition can be eliminated and the call to `fail-now` can be moved to the end of the function after the `for` loop
2. It would be helpful to print the expected and actual count of SVIDs in the cache for debugging